fireapp/lib/presentation/constraint_form/constraint_form_view.dart

```diff
@@ -0,0 +1,132 @@
+import 'package:fireapp/presentation/constraint_form/time_input_field.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:fireapp/presentation/constraint_form/constraint_form_view_model.dart';
+import 'package:fireapp/presentation/constraint_form/base_input_field.dart';
+
+class SchedulerConstraintPage extends StatelessWidget {
+  const SchedulerConstraintPage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(title: const Text('Scheduler')),
+      resizeToAvoidBottomInset: false,
+      body: const SchedulerConstraintForm(),
+    );
+  }
+}
+
+class SchedulerConstraintForm extends StatefulWidget {
+  const SchedulerConstraintForm({super.key});
+
+  @override
+  _SchedulerConstraintFormState createState() => _SchedulerConstraintFormState();
+}
+
+class _SchedulerConstraintFormState extends State<SchedulerConstraintForm> {
+  final viewModel = SchedulerConstraintFormViewModel();
+
+  @override
+  Widget build(BuildContext context) {
+    return Form(
+      key: viewModel.formKey,
+      child: Padding(
+        padding: const EdgeInsets.all(12.0),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            SchedulerTextInputField(
+              controller: viewModel.titleController,
+              style: Theme.of(context)
+                  .textTheme
+                  .labelLarge
+                  ?.copyWith(color: Colors.grey[700]),
+              label: AppLocalizations.of(context)?.volunteer_name ??
+                  "", // This will show volunteer name
+              icon: Icons.title,
+              validator: (v) => v!.isEmpty ? 'Title is empty!' : null,
+            ),
+            const SizedBox(height: 2.0),
+            Row(
+              children: [
+                Expanded(
+                  child: DropdownButtonFormField<int>(
+                    decoration: InputDecoration(
+                      labelStyle: Theme.of(context)
+                          .textTheme
+                          .labelLarge
+                          ?.copyWith(color: Colors.grey[700]),
+                      filled: true,
+                      fillColor: Colors.white,
+                      prefixIcon: const Icon(Icons.arrow_drop_down),
+                      labelText: AppLocalizations.of(context)?.selectAsset,
+                      border: BaseInputField.commonInputBorder,
+                    ),
+                    value: viewModel.dropdownValue,
+                    items: <int>[1, 2, 3, 4]
+                        .map<DropdownMenuItem<int>>((int value) {
+                      return DropdownMenuItem<int>(
+                        value: value,
+                        child: Text(
+                          'Asset $value',
+                          style: Theme.of(context).textTheme.labelLarge?.copyWith(
```

Comment:
Do not copy the theme with hard coded colours. You should be using colours defined in the theme. Find the closest colour or add it to the theme